pkg/rehearse/jobs.go

```diff
@@ -42,7 +42,8 @@ const (
 	logCiopConfigFile            = "ciop-config-file"
 	logCiopConfigRepo            = "ciop-config-repo"
 
-	clusterTypeEnvName = "CLUSTER_TYPE"
+	clusterTypeEnvName  = "CLUSTER_TYPE"
+	canBeRehearsedLabel = "pj-rehearse.openshift.io/can-be-rehearsed"
```

Comment:
Can this be shared and imported?

Reply:
yea, it turns out that we need that for the tests as well.

Reply:
So actually in the test file I would not use the import -- we want someone who changes the value of a constant like this to have tests break so they know they're changing functionality. But sharing this between rehearse and gen is good